framework/src/wifi_manager/wifi_manager_utils.h

```diff
@@ -72,6 +72,18 @@ int net_task_create(FAR const char *name, int priority, int stack_size,
 		}												\
 	} while (0)
 
+/* errno EINVAL shouldn't be happened
+ * if it were happened then system could be corrupted */
```

Comment:
Could you please change "errno EINVAL shouldn't be happened * if it were happened then system could be corrupted" to "errno EINVAL shouldn't happen * if it happens then the system could be corrupted"

Reply:
Thanks I'll modify